src/app/app.component.spec.ts

```diff
@@ -25,11 +26,4 @@ describe('AppComponent', () => {
     const app = fixture.debugElement.componentInstance;
     expect(app.title).toEqual('myapp');
   });
-
-  it('should render title', () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    fixture.detectChanges();
-    const compiled = fixture.debugElement.nativeElement;
-    expect(compiled.querySelector('.content span').textContent).toContain('myapp app is running!');
-  });
```

Comment:
This test was no longer needed since the app doesn't render "myapp app is running" anywhere.